api/classes/database.js

```diff
@@ -0,0 +1,16 @@
+const config = require('../../knexfile')['development'];
+
+// reset database at each startup
+//try {
```

Comment:
I've been taught to hate commented code with violent prejudice but I can live with this ;)